CharlieBackend.Business/Services/LessonService.cs

```diff
@@ -168,5 +198,32 @@ public async Task<Result<IList<StudentLessonDto>>> GetStudentLessonsAsync(long s
             return Result<IList<StudentLessonDto>>.GetSuccess( _mapper.Map<IList<StudentLessonDto>>(studentLessonModels));
         }
 
+        private async Task<bool> CheckStudentsID(IList<VisitDto> visit, long groupId)
+        {
+            List<long> studentsId = new List<long>();
+
+            foreach (var item in visit)
+            {
+                studentsId.Add((long)item.StudentId);
+            }
+
+            var checkStudentInGroup = await _unitOfWork.StudentGroupRepository.GetAllStudentInGroup(groupId);
+
+            if (checkStudentInGroup.Count != studentsId.Count)
+            {
+                return false;
+            }
+
+            foreach (var item in studentsId)
```

Comment:
Why do you need this check?
Seems like it always be true.

Reply:
it is possible that the  mentor has writed wrong StudentId or  student may not be  included in group on this lesson